apps/precursor-chemicals/data/chemicals.json

```diff
@@ -0,0 +1,212 @@
+    "value": "Alpha-phenylacetoacetonitrile (APAAN)",
+    "category": "1",
+    "cnCode": "2926 4000"
+  },
+  {
+    "label": "Chloroephedrine (2939 7990)",
+    "value": "Chloroephedrine",
+    "category": "1",
+    "cnCode": "2939 7990"
+  },
+  {
+    "label": "Chloropseudoephedrine (2939 7990)",
+    "value": "Chloropseudoephedrine",
+    "category": "1",
+    "cnCode": "2939 7990"
+  },
+  {
+    "label": "Ephedrine (2939 4100)",
+    "value": "Ephedrine",
+    "category": "1",
+    "cnCode": "2939 4100"
+  },
+  {
+    "label": "Ergometrine (2939 6100)",
+    "value": "Ergometrine",
+    "category": "1",
+    "cnCode": "2939 6100"
+  },
+  {
+    "label": "Ergotamine (2939 6200)",
+    "value": "Ergotamine",
+    "category": "1",
+    "cnCode": "2939 6200"
+  },
+  {
+    "label": "Isosafrole (2932 9100)",
+    "value": "Isosafrole",
+    "category": "1",
+    "cnCode": "2932 9100"
+  },
+  {
+    "label": "Lysergic Acid (2939 6300)",
+    "value": "Lysergic Acid",
+    "category": "1",
+    "cnCode": "2939 6300"
+  },
+  {
+    "label": "N-acetylanthranilic Acid (2924 2300)",
+    "value": "N-acetylanthranilic Acid",
+    "category": "1",
+    "cnCode": "2924 2300"
+  },
+  {
+    "label": "Norephedrine (2939 4400)",
+    "value": "Norephedrine",
+    "category": "1",
+    "cnCode": "2939 4400"
+  },
+  {
+    "label": "Piperonal (2932 9300)",
+    "value": "Piperonal",
+    "category": "1",
+    "cnCode": "2932 9300"
+  },
+  {
+    "label": "Pseudoephedrine (2939 4200)",
+    "value": "Pseudoephedrine",
+    "category": "1",
+    "cnCode": "2939 4200"
+  },
+  {
+    "label": "Safrole (2932 9400)",
+    "value": "Safrole",
+    "category": "1",
+    "cnCode": "2932 9400"
+  },
+  {
+    "label": "4-anilino-N-phenethylpiperidine (ANPP) (2933 3999)",
+    "value": "4-anilino-N-phenethylpiperidine (ANPP)",
+    "category": "1",
+    "cnCode": "2933 3999"
+  },
+  {
+    "label": "N-phenethyl-4-piperidone (NPP) (2933 3999)",
+    "value": "N-phenethyl-4-piperidone (NPP)",
+    "category": "1",
+    "cnCode": "2933 3999"
+  },
+  {
+    "label": "3-(1,3-benzodioxol-5-yl)-2-methyloxirane-2-carboxylate (PMK methyl glycidate) (2932 9900)",
+    "value": "3-(1,3-benzodioxol-5-yl)-2-methyloxirane-2-carboxylate (PMK methyl glycidate)",
+    "category": "1",
+    "cnCode": "2932 9900"
+  },
+  {
+    "label": "3-(1,3-benzodioxol-5-yl)-2-methyloxirane-2-carboxylic acid (PMK glycidic acid) (2932 9900)",
+    "value": "3-(1,3-benzodioxol-5-yl)-2-methyloxirane-2-carboxylic acid (PMK glycidic acid)",
+    "category": "1",
+    "cnCode": "2932 9900"
+  },
+  {
+    "label": "alpha-phenylacetoacetamide (APAA) (2924 2970)",
+    "value": "alpha-phenylacetoacetamide (APAA)",
+    "category": "1",
+    "cnCode": "2924 2970"
+  },
+  {
+    "label": "methyl alpha-phenylacetoacetate (MAPA) (2918 3000)",
+    "value": "methyl alpha-phenylacetoacetate (MAPA)",
+    "category": "1",
+    "cnCode": "2918 3000"
```

Comment:
I know the table doesn't have these capitalised but I think these two chemicals should be capitalised like the others so its consistent